### ISSUE TITLE
Chybějící nebo špatně pojmenované hodnoty/kóty

### ISSUE THREAD

**Issue by tuxj** [3DprintFIT/B182HW-Assignment#11]
Ve výkresu je kóta průměru odsazení středů matiček jako tr_nut_mounting_holes, nicméně v zadání tato proměnná není, jediná podobná je tr_nut_mount_hole. Je to ta správná?

To samé:
bolt_hole X bolt_hole_diameter
x_smooth_rods_distance X x_smooth_rod_distance

Dále mám dotaz: nemůžu přijít na to, jestli nechybí kóta rádiusu (nebo poloměru) u zakulacené strany základny (u matičky) nebo jen jestli bych to měl vědět a málo hledám/nad tím přemýšlím...

Díky

**Comment by marekreimer** [comment]
tr_nut_mounting_holes == tr_nut_mount_hole
bolt_hole == bolt_hole_diameter
x_smooth_rods_distance == x_smooth_rod_distance

Vsechno to jsou preklepy.

Kota radiusu u zakulacene hrany zakladny nechybi, stred tohohle zaobleni je shodny se stredem diry na sroub ktery drzi trapezovou matici. Na obrazku vyznaceny cervene:
![detail](https://user-images.githubusercontent.com/32454894/58418317-c8d73c80-8087-11e9-9f47-62b3d0d5a334.PNG)

@hroncok Mam to prejmenovat ve vykresech nebo v zadani a testech nebo vubec?

**Comment by tuxj** [comment]
Ano, střed chápu, ~~nicméně tím způsobem, jakým to tvořím (sloučením válce a dvou kvádrů) potřebuji znát průměr/poloměr této zakulacené hrany a nemůžu se k němu dopočítat :/ (nebo nevím, jak zjistit pozici) a dohromady s (nepřesnou) neznalostí celkového X rozměru je celkem hell... nebo na to jdu úplně blbě?~~

měl jsem celou linear_bearing_hole shiftlou, proto mi to nevycházelo (už jsem se dopočítal)
Díky
